src/app/util/attribute-storage.cpp

```diff
@@ -109,6 +109,10 @@ const uint16_t attributeManufacturerCodeCount                   = GENERATED_ATTR
 #define endpointNetworkIndex(x) fixedNetworks[x]
 #endif
 
+namespace {
+app::AttributeAccessInterceptor * attributeAccessInterceptors = nullptr;
```

Comment:
```suggestion
app::AttributeAccessInterceptor * gRegisteredAttributeAccessors = nullptr;
```

Reply:
Adding the "g" makes sense.  For the rest, again it's not just "accessors"...  you can't get a "full" picture of attributes from those objects. They are a part of the pipeline, not the whole thing.

Reply:
Renamed to `gAttributeAccessOverrides`.